app/models/user_dashboard.rb

```diff
@@ -28,7 +28,7 @@ def email
   end
 
   def used_space_percent
-    "25%"
+    @_user.storage_used
```

Comment:
@david Can't wait to see the progress bar changing as we work with the content! :-) We should refactor: instead of referring to `@_user` we should call `user`. I will cover that under `lb-sweeper-3`.